Podfile

```diff
@@ -166,7 +166,7 @@ abstract_target 'Apps' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.61.0'
+    gutenberg :tag => 'v1.61.1'
```

Comment:
Note: I checked and `develop` is currently on `gutenberg :tag => 'v1.61.0'`, so making `1.61.1` the one landing in `develop` after the merge seems correct to me 👍
(as opposed to having already a newer version in `develop` like it sometimes happens)